src/auth/auth.action.js

```diff
@@ -51,14 +53,24 @@ export const getUser = () => {
 export const getOrgs = () => {
   return (dispatch, getState) => {
     const accessToken = getState().auth.accessToken;
+    const login = getState().auth.user.login;
 
     dispatch({ type: GET_AUTH_ORGS.PENDING });
 
-    fetchAuthUserOrgs(accessToken)
+    Promise.all([
+      fetchAuthUserOrgs(accessToken),
+      fetchUserOrgs(login, accessToken),
+    ])
       .then(data => {
+        const orgs = [];
+
+        data.forEach(part => part.map(org => orgs.push(org)));
+
         dispatch({
           type: GET_AUTH_ORGS.SUCCESS,
-          payload: data,
+          payload: uniqby(orgs, 'id').sort(
+            (org1, org2) => org1.login > org2.login
+          ),
```

Comment:
GitHub sorts orgs on profiles by the creation date of the organization. I like alphabetically more, but I'm not sure if anyone feels strongly about trying to mimic GitHub?